### PR TITLE
Gnome 45 support

### LINKED ISSUE #59: Extensión doesn't work in GNOME 45
In GNOME 45 (Ubuntu 23.10) the extension doesn't start showing this error in the journal:
```
JS ERROR: Extension notifications-alert-on-user-menu@hackedbellini.gmail.com: SyntaxError: import declarations may only appear at top level of a module @ resource:///org/gnome/shell/ui/main.js:3:0
```

### PR DESCRIPTION
**Pull request by zweif**
This works for me on Gnome 45 (Ubuntu 23.10).
Fixes #59

### CODE REVIEW

File: src/locale/pt_BR.po [hunk 1]
```diff
@@ -17,100 +17,113 @@ msgstr ""
...
 
-#: src/prefs.js:119
+#: src/prefs.js:114
 msgid "Choose an application to blacklist:"
 msgstr "Escolha um aplicativo para adicionar na lista negra:"
 
-#: src/prefs.js:44
+#: src/prefs.ui:6
+msgid "Colors"
+msgstr ""
```
Comment (bellini666) [inline src/locale/pt_BR.po:68]:
```suggestion
msgstr "Cores"
```

File: src/locale/pt_BR.po [hunk 2]
```diff
@@ -17,100 +17,113 @@ msgstr ""
...
+
+#: src/prefs.ui:107
+#, fuzzy
+msgid "Filter"
+msgstr "Lista do filtro"
```
Comment (bellini666) [inline src/locale/pt_BR.po:73]:
```suggestion
msgid "Filter"
msgstr "Filtro"
```

File: src/locale/pt_BR.po [hunk 3]
```diff
@@ -17,100 +17,113 @@ msgstr ""
...
 
-#: src/prefs.js:80
+#: src/prefs.js:75
 msgid "Application"
 msgstr "Aplicativo"
 
-#: src/prefs.ui:214
+#: src/prefs.ui:59
+msgid "Behavior"
+msgstr ""
```
Comment (bellini666) [inline src/locale/pt_BR.po:48]:
```suggestion
msgstr "Comportamento"
```

**Review by bellini666 — CHANGES_REQUESTED** [review]
I added suggestions for the pt_BR translations, as I'm brazilian =P

But for the other languages, probably it is better to remove the empty strings so at least the original english strings will be used

**Comment by zweif** [comment]
It seems that my wording was misleading. If the translation contains an empty string, the original one is used instead. I wanted to say that the translations are incomplete. Removing the empty strings would likely interfere with tools for translation like Poedit.

Would you do the changes to pt_BR.po?

**Comment by bellini666** [comment]
Oh, I see. Didn't know that =P

Will merge this then the way it is and try to release a new version on ego :)

**Pull request merged**